Docs/Configuration.tex

```diff
@@ -2230,8 +2229,8 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
     \item Download \href{https://github.com/LongSoft/UEFITool/releases}{UEFITool} and
       \href{https://github.com/LongSoft/Universal-IFR-Extractor/releases}{IFR-Extractor}.
     \item Open the firmware image in UEFITool and find \texttt{CFG Lock} unicode string.
-      If it is not present, the firmware may not have this option and one should stop.
-    \item Extract the \texttt{Setup.bin} PE32 Image Section (the one UEFITool found) through
+      If it is not present, the firmware may not have this option and the proccess therefore be discontinued.
```

Comment:
the process *should* therefore be discontinued I guess?

Reply:
Noted